keyboards/macroflow_original/readme.md

```diff
@@ -0,0 +1,27 @@
+# macroflow_original
+
+![Imgur](https://i.imgur.com/hIytJTl.png)
+
+*A 2x4 macropad with a built in encoder and OLED screen*
+
+* Keyboard Maintainer: [Patrickemm](https://github.com/Patrickemm)
+* Hardware Supported: *ATmega32U4*
+* Hardware Availability: *https://keeb.io/products/pro-micro-usb-c-version-5v-16mhz-arduino-compatible-atmega32u4*
```

Comment:
It would be appreciated if you could provide a link to where keyboard-specific information could be found under the "Hardware Availability" section, like a Github link to open source files, etc.
```suggestion
* Hardware Supported: Pro Micro compatible development board
* Hardware Availability: ...
```

Reply:
Could I add a link to the product page for the macropad? (after it is up for sale)

Reply:
Yep, that'd be great.

Reply:
And it is okay that I can add that link after the macropad is finished including the qmk merge?

Reply:
Yes, it can be updated if needed. But if you are able to predict/know the URL, it would be easiest to add it to this PR, even if it is not up yet.

Reply:
All I could do is put a link to the eBay store that it will be posted on.